.github/workflows/run-test-cases.yml

```diff
@@ -229,10 +229,11 @@ jobs:
           kubeadm version && echo "kubeadm return code: $?" || echo "kubeadm return code: $?"
           kubelet --version && echo "kubelet return code: $?" || echo "kubelet return code: $?"
           sudo swapoff -a              
-          sudo kubeadm init
+          sudo kubeadm init --pod-network-cidr=10.244.0.0/16
```

Comment:
any reason why not rely on the default?

Reply:
K8s [docs](https://v1-20.docs.kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#pod-network) say "Take care that your Pod network must not overlap with any of the host networks: you are likely to see problems if there is any overlap. (If you find a collision between your network plugin's preferred Pod network and some of your host networks, you should think of a suitable CIDR block to use instead, then use that during kubeadm init with --pod-network-cidr and as a replacement in your network plugin's YAML)."

Just using the default without a `pod-network-cidr` caused a `error: /run/flannel/subnet.env: no such file or directory` error in kubelet during Pod bringup. This [issue suggests](https://github.com/kubernetes/kubernetes/issues/70202#issuecomment-485145431) adding the CIDR network resolves it. We shall see.